core/storage/exploration/gae_models.py

```diff
@@ -601,6 +601,13 @@ class StateIdMappingModel(base_models.BaseModel):
     # from this value + 1.
     largest_state_id_used = ndb.IntegerProperty(indexed=True, required=True)
 
+    @staticmethod
+    def get_deletion_policy():
```

Comment:
I think it's already deleted, actually. @aks681 @nithusha21 can you confirm? Should we get rid of it from the codebase, too?

Reply:
Yeah, the state ID mapping models are deleted from prod and test server.

Reply:
Yes these are deleted completely. I will make a PR that removes it from the code too!

Reply:
https://github.com/oppia/oppia/pull/7617 This is the PR I created.